components/kfp/Run_component/component.py

```diff
@@ -0,0 +1,47 @@
+from typing import NamedTuple
+
+
+def run_component_or_pipeline(
+    component_url: 'Url',
+    arguments: dict,
+    endpoint: str = None,
+    wait_timeout_seconds: float = None,
+) -> NamedTuple('Outputs', [
+    ('run_id', str),
+    ('run_object', 'JsonObject'),  # kfp.ApiRunDetails
+]):
+    import json
+    import os
+    import kfp
+    from kfp_server_api import ApiClient
+    print('Loading component...')
+    op = kfp.components.load_component_from_url(component_url)
+    print('Loading component done.')
+    print('Submitting run...')
+    if not endpoint:
+        endpoint = os.environ['ML_PIPELINE_SERVICE_HOST'] + ':' + os.environ['ML_PIPELINE_SERVICE_PORT']
+    create_run_result = kfp.Client(host=endpoint).create_run_from_pipeline_func(op, arguments=arguments)
+    run_id = str(create_run_result.run_id)
+    print('Submitted run: ' + run_id)
+    print('Waiting for the run to finish...')
+    run_object = create_run_result.wait_for_run_completion(wait_timeout_seconds)
+    print('Run has finished.')
+    # sanitize_for_serialization uses correct field names and properly converts datetime values
+    run_dict = ApiClient().sanitize_for_serialization(run_object)
```

Comment:
what does this do?

Reply:
This is basically `.to_dict()` that works properly (`camelCase`, not `snake_case`). I thought the issue was fixed in the Swagger/OpenAPI generator, but maybe has it resurfaced.

Reply:
I'm a little confused, is `camelCase` the expected output? why?

Reply:
That's what Kubernetes REST API expects. AFAIK, the server will reject camel_case attributes.
But some Swagger generated clients have a bug and uses python names in `.to_dict()`.